semantic.cabal

```diff
@@ -340,6 +340,7 @@ test-suite test
   import:              haskell, dependencies, executable-flags
   type:                exitcode-stdio-1.0
   hs-source-dirs:      test
+  ghc-options:         -Wunused-imports
```

Comment:
Can we apply this in a common stanza?

Reply:
Yup. We should really just turn on -Wall everywhere, but I’ll file a separate issue about that.

Reply:
I dig it; the thing `fused-effects` does is working pretty nicely.